### PR TITLE
Create 4-5.md

### PR DESCRIPTION
**Pull request by claperle**

### CODE REVIEW

File: docs/section4/4-5.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+**Appendix G Section:** Section G3.1-4 exception #2 Schedules for the proposed building  
+**Appendix G Section Reference:**  
+
+**Applicability:** All required data elements exist for P_RMR  
+**Applicability Checks:** 
+
+1. P-PRM is modeled with a least one space with health and safety mandated minimum ventilation requirements during unoccupied hours. 
+
+**Manual Check:** None  
+**Evaluation Context:** Evaluate each data element
```
Comment (mkarpman) [inline docs/section4/4-5.md:16]: Same comment as for 4-3

File: docs/section4/4-5.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+**Manual Check:** None  
+**Evaluation Context:** Evaluate each data element  
+**Data Lookup:** None  
+**Function Call:** None
+
+## Rule Logic:
+- For each zone in P_RMR: `for zone_p in P_RMR...zones:`
+    - For each space in zone: `for space_p in zone_p.zones:`
+        - Check if space(s) is of type that has health and safety mandated minimum ventilation requirements during unoccupied hours: `if space_p.lighting_space_type in [CORRIDOR_HOSPITAL,LABORATORY_EXCEPT_IN_OR_AS_A_CLASSROOM,HEALTHCARE_FACILITY_OPERATING_ROOM]:`
```
Comment (mkarpman) [inline docs/section4/4-5.md:23]: I am not sure hospital corridors are subject to the exception. Did we plan to run it by Mike T? Should all lighting space types applicable to Healcare Facility be included?

File: docs/section4/4-5.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+            - Set applicability flag: `rule_applicability_check = TRUE`
+                - For each terminal in zones that meet the applicabliity check: `for terminal_p in zone_p.terminals:`
+                    - Get HVAC system serving terminal: `hvac_p = terminal_p.served_by_heating_ventilation_air_conditioning_systems`
+                        - Get fan system of HVAC system: `fan_system_p = hvac_p.fan_systems`
+                        -   Get fan system operating schedule: `operating_schedule_p = fan_system_p.operating_schedule`
+
+  **Rule Assertion:**
+  - Case 1: For each applicable fan system the fan schedule is 1 for all hour: `if operating_schedule_p[i] == 1 for i in range(8760): PASS`
+
```
Comment (mkarpman) [inline docs/section4/4-5.md:32]: I think the check for this RDS should be the same as for 4-3 and 4-4, to look at operation_during_occupied and operation_furing_unoccupied.

File: docs/section4/4-5.md (hunk 2 above)
Comment (claperle) [reply docs/section4/4-5.md:23]: I emailed him and he never responded. I do not think all healthcare should be included but I don't know these rules so this opinion is of little value lol.

File: docs/section4/4-5.md (hunk 1 above)
Comment (claperle) [reply docs/section4/4-5.md:16]: Occupied hours is covered by 4-3.

File: docs/section4/4-5.md (hunk 3 above)
Comment (claperle) [reply docs/section4/4-5.md:32]: Great, that is much simpler. Update made.

File: docs/section4/4-5.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+        - For each terminal in zone_p: `for terminal_p in zone_p.terminals:`
+            - Get HVAC system serving terminal: `hvac_p = terminal_p.served_by_heating_ventilation_air_conditioning_systems`
+            - Add to list of heating_ventilation_air_conditioning_systems_p as the code loops through the terminal units: `heating_ventilation_air_conditioning_systems_list_p = heating_ventilation_air_conditioning_systems_list_p.append(heating_ventilation_air_conditioning_systems_p)`                    
+        - Add to master list of HVAC system that are applicable to this zone after looping through the terminal units: `applicable_hvac_systems_list_p = applicable_hvac_systems_list_p.append(heating_ventilation_air_conditioning_systems_list_p)`         
+- Convert the list of applicable_hvac_systems_list_p to a set and the back to a list to eliminate duplicates after looping through all zones: `applicable_hvac_systems_list_p = list(set(applicable_hvac_systems_list_p))`                             
+- For each hvac_p in applicable_hvac_systems_list_p: `for each hvac_p in applicable_hvac_systems_list_p:`
```
Comment (mkarpman) [inline docs/section4/4-5.md:35]: 1. I wonder if we can replace the above by just saying to locate all HeatingVentilationAirConditioningSystems that serve at least one zone returned by get_zones_health_safety_vent_reqs(P_RMR).
2. I believe the rule for continuous operation only applies to HVAC systems that serve applicable zones AND provide ventilation, so I think we also need to add a check for the corresponding FanSystem.minimum_outdoor_airflow>0

Comment (supriyagoel) [reply docs/section4/4-5.md:35]: good point about checking for only systems that provide ventilation.

Comment (supriyagoel) [reply docs/section4/4-5.md:35]: as discussed on 2022.03.16 RDS call, will create a function to get HVAC systems based on zone list and use that here.

File: docs/section4/4-5.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+- For each hvac_p in applicable_hvac_systems_list_p: `for each hvac_p in applicable_hvac_systems_list_p:`                         
+    - Get fan operation during unoccupied periods: `operation_during_unoccupied_p = hvac_p.fan_systems[0].operation_during_unoccupied`  
+    - Get minimum_outdoor_airflow: `minimum_outdoor_airflow_p = hvac_p.fan_systems[0].minimum_outdoor_airflow`  
+    **Rule Assertion:**
+    - Case 1: For each applicable hvac system, if fan system operation during unoccupied periods is continuous in the P_RMR: `if operation_during_unoccupied_p == “CONTINUOUS” and minimum_outdoor_airflow_p > 0: PASS`
+    - Case 2: Else, for each applicable fan system,:`Else: UNDETERMINED and raise_message "THE HVAC SYSTEM SERVES ZONE(S) THAT APPEAR LIKELY TO HAVE HEALTH AND SAFETY MANDATED MINIMUM VENTILATION REQUIREMENTS DURING UNOCCUPIED HOURS AND THEREFORE MAY WARRANT CONTINUOUS OPERATION DURING UNOCCUPIED HOURS PER SECTION G3.1-4 SCHEDULES EXCEPTION #2 FOR THE PROPOSED BUILDING." `
+
```
Comment (supriyagoel) [inline docs/section4/4-5.md:37]: question for @mkarpman should case 2 be fail instead on undetermined?

Comment (mkarpman) [reply docs/section4/4-5.md:37]: I agree.

Comment (claperle) [reply docs/section4/4-5.md:32]: Okay it has been changed.

**Review by supriyagoel — APPROVED** [review]

**Pull request merged**